Update GOCART settings for p8/p81

bbakernoaa opened the pull request:
This PR updates the GOCART settings for p8/p81 and should be done after [PR#795](https://github.com/NOAA-EMC/global-workflow/pull/795) 

It addresses #816 updating the default wet scavenging coefficients when gocart is enabled.

It addresses #814 by fixing a bug in which when running only ATM-AERO simulations the value from parm/config.aero was not read and therefore defaults to zero.

It addresses #817 by updating the dust inputs for the fengsha dust emission scheme.  These updates include a newer soil database (SOILGRIDSv2) and updates for the drag partition and threshold velocity.  It also appropriately scales the emissions by modifying the alpha value found in the dust component configuration file.

It addresses #815.  With @rmontuoro taking his new position the default location of the GOCART emission files needs to be modified so that the aerosol group can modify the input emissions as necessary for development.  

I tested this on HERA using an ATM-AERO only configuration.  

- [x] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested?**

- [x] Forecast-only test on Hera
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published


@zhanglikate @gjfrost

ChunxiZhang-NOAA committed:
Updated the geographical static data, ICs, SDFs and namelist settings in CCPP for p8c

ChunxiZhang-NOAA committed:
Merged with the remote upstream/develop into the feature/p8c branch

ChunxiZhang-NOAA committed:
Updated namelists for the RRTMGP scheme and updated diag_table pwatclm->pwat

bbakernoaa committed:
Merge remote-tracking branch 'CZ/feature/p8c' into feature/p8c_aero

bbakernoaa committed:
Adding changes for the dust inputs and scaling factor

bbakernoaa committed:
Update convective wet scavenging factors

bbakernoaa committed:
Fix fscav when not fully coupled - ie atm-aero only

bbakernoaa committed:
Update Emission source location for HERA

With Raffaele leaving we are moving the location of the emission data
to another group members location on hera.  This is to make sure that
the AEROSOL group retains control of the emission data and can modify
the directories with any changes for the future.

bbakernoaa committed:
Fix ExtData path for dust inputs

bbakernoaa commented:
Addressing #814, @zhanglikate performed a test with this fix and it works properly. Please see the attached presentation paying attention to `v174` (without the fix) and `v175` with the fix. 

https://docs.google.com/presentation/d/1pIu4AIDKL6zfksOkAmeNhAN1WOQfcGl1/edit?usp=sharing&ouid=111407383710090576995&rtpof=true&sd=true

WalterKolczynski-NOAA committed:
Merge branch 'develop' of github.com:NOAA-EMC/global-workflow into feature/p8c_aero

WalterKolczynski-NOAA commented on parm/chem/ExtData.other:
```diff
@@ -11,12 +11,12 @@ TROPP            'Pa'    Y       N                  -            0.0      1.0
-DU_SSM           '1'  Y E %y4-%m2-%d2T12:00:00 none none ssm        ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_RDRAG         '1'  Y E           -          none none drag_part  ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
-DU_UTHRES        '1'  Y E           -          none none uthres     ExtData/Dust/FENGSHA_SOILGRIDS2017_GEFSv12_v1.0.nc
+DU_CLAY           '1'  Y E           -           none none clayfrac    ExtData/dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
```
This path changed the uppercase 'D' in Dust into lowercase, but the directory names didn't actually change.

bbakernoaa replied:
You are right the filenames should have changed here.  

They should be 

```bash
DU_CLAY           '1'  Y E           -           none none clayfrac    ExtData/Dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
DU_SAND           '1'  Y E           -           none none sandfrac    ExtData/Dust/FENGSHA_SOILGRIDS2019_GEFSv12_v1.2.nc
DU_SILT           '1'  Y E           -           none none siltfrac    /dev/null
DU_SSM            '1'  Y E           -           none none ssm         /dev/null:1.0
DU_RDRAG          '1'  Y E %y4-%m2-%d2t12:00:00  none none albedo_drag ExtData/Dust/FENGSHA_Albedo_drag_v1.nc
DU_UTHRES         '1'  Y E           -           none none uthres      ExtData/Dust/randomforestensemble_uthres.nc
```

And the path that you changed the ExtData to point to is not synced with that path that was there. I don't understand why this was modified when it was fine with pointing to Raffaele's directory.  It should continue to point to a location that the aerosol team has access to. 


You changed it from here `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/NASA` to `/scratch2/NCEPDEV/ensemble/noscrub/Walter.Kolczynski/global-workflow/gocart_emissions/Dust/`

WalterKolczynski-NOAA reviewed (changes requested):
I don't think this actually addresses #815 yet, as there are no changes to AERO_INPUTS_DIR in config.aero. I'm copying Rafaelle's directory to a new location under my account; I'll push an update in the morning after it completes.

WalterKolczynski-NOAA committed:
Update aerosol external data path on Hera

Rafaelle moved into a management position, so we need to move the external
aerosol data to a new location. Placed it in my personal space like the
other two machines for the time being. All should be moved to glopara
space at some point.

Fixes #815

bbakernoaa commented:
@WalterKolczynski-NOAA The `AERO_INPUTS_DIR` should point to a location someone in the Aerosol team has access to. 

It should be here `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/NASA/ExtData`

Why is it arbitrarily set to your directory instead of the one above?

bbakernoaa commented:
These has the latest updates.  Raffaele's directory does not

bbakernoaa committed:
Merge branch 'feature/p8c_aero' of https://github.com/bbakernoaa/global-workflow into feature/p8c_aero

bbakernoaa committed:
updating dust

WalterKolczynski-NOAA commented:
> @WalterKolczynski-NOAA The `AERO_INPUTS_DIR` should point to a location someone in the Aerosol team has access to.
> 
> It should be here `/scratch1/RDARCH/rda-arl-gpu/Barry.Baker/emissions/NASA/ExtData`
> 
> Why is it arbitrarily set to your directory instead of the one above?

When I changed it, it hadn't been changed away from Rafaelle's directory at all. I think there is also value in separating what the aerosol team is working on from what is officially supporting the workflow. I can update the "official" version whenever requested, but it would be less volatile than pointing to an active development copy, particularly since I also have to sync changes to other machines. Other aspects of global workflow (like the fix directory) work the same way.

Really it should be held under the global group account instead of my personal account, but I somehow still don't have access to that on all machines.

bbakernoaa commented:
@WalterKolczynski-NOAA if it isn't going to be put under the global group account then I don't see why it can't be put under may account space.  This just seems like an arbitrary decision.

WalterKolczynski-NOAA commented:
> @WalterKolczynski-NOAA if it isn't going to be put under the global group account then I don't see why it can't be put under may account space. This just seems like an arbitrary decision.

@bbakernoaa: @KateFriedman-NOAA has copied the data to `/scratch1/NCEPDEV/global/glopara/data/gocart_emissions` on Hera. When we put the data on WCOSS2, we'll also use glopara there. Not going to bother moving the data on Dell since it only has a month of life left. On Orion there is no glopara, so the current location under me is just as good as one under Kate.

bbakernoaa committed:
Merge branch 'NOAA-EMC:develop' into feature/p8c_aero

WalterKolczynski-NOAA committed:
Update Hera aerosol emission location to glopara

Refs: #815
